### PR TITLE
SC-18979 release notes 3.4.1

### PR DESCRIPTION
**Pull request by yago-123**

### CODE REVIEW

File: docs/agents/sematext-agent/releasenotes.md [hunk 1]
```diff
@@ -39,6 +39,20 @@ NULL
 
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
+
+- Support for monitoring services running on the Containerd runtime in Kubernetes has been added.
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:48]: running on the ==> using

**Review by otisg — COMMENTED** [review]
LGTM

File: docs/agents/sematext-agent/releasenotes.md [hunk 2]
```diff
@@ -39,6 +39,20 @@ NULL
...
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than regular container billing.
+- Enhanced the reliability of the Kubernetes initialization structure.
+
+### Bug Fixes
+
+- Resolved a service discovery edge-case bug occurring during cold starts in ElasticSearch for Kubernetes and Docker environments.
```
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:54]: Let's remove this one and mention only ESv8.
```
- Resolved a service discovery bug occurring with ElasticSearch v8.
```

File: docs/agents/sematext-agent/releasenotes.md [hunk 3]
```diff
@@ -39,6 +39,20 @@ NULL
 
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
```
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:46]: You may want to add this as well;
```
- Corrected inaccurate information about agent_started events in Sematext Cloud that gave the impression the agent was frequently restarting.
```

File: docs/agents/sematext-agent/releasenotes.md [hunk 4]
```diff
@@ -39,6 +39,20 @@ NULL
...
+
+- Support for monitoring services running on the Containerd runtime in Kubernetes has been added.
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than regular container billing.
+- Enhanced the reliability of the Kubernetes initialization structure.
+
+### Bug Fixes
+
+- Resolved a service discovery edge-case bug occurring during cold starts in ElasticSearch for Kubernetes and Docker environments.
+
```
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:55]:
```
- Fixed a bug that affected reading Nginx configuration files, to able to discover your `server_status` path.
```

File: docs/agents/sematext-agent/releasenotes.md (hunk 1 above)
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:48]: Is this about the JMX issue? If yes, the text is TOO generic, it sounds like we start supporting containerd today

File: docs/agents/sematext-agent/releasenotes.md (hunk 4 above)
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:55]:
```
- Resolved an edge-case bug occurring during discovering services via eBPF, that leads to have incorrect service IP.
```

**Review by boratanrikulu — CHANGES_REQUESTED** [review]
see the comments

File: docs/agents/sematext-agent/releasenotes.md (hunk 2 above)
Comment (yago-123) [reply docs/agents/sematext-agent/releasenotes.md:54]: I actually copied what you wrote in the previous release just to be explicit about the new envs, but OK, modifying

File: docs/agents/sematext-agent/releasenotes.md [hunk 5]
```diff
@@ -39,6 +39,20 @@ NULL
...
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
+
+- Support for monitoring services running on the Containerd runtime in Kubernetes has been added.
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than regular container billing.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:49]: regular -> you mean existing?

**Review by cpipilas — APPROVED** [review]
LGTM, only a minor comment added

File: docs/agents/sematext-agent/releasenotes.md [hunk 6]
```diff
@@ -39,6 +39,23 @@ NULL
...
+### Improvements
+
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than existing container billing.
+- Enhanced the reliability of the Kubernetes initialization structure.
+- Corrected inaccurate information about `agent_started` events in Sematext Cloud that gave the impression the agent was frequently restarting.
+
+### Bug Fixes
+
+- Resolved a service discovery bug occurring with ElasticSearch version 8.
+- Fixed a bug that affected reading Nginx configuration files, to able to discover your `server_status` path.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:55]: to able to discover your server_status path ->  to enable discovery of your `server_status` path.

File: docs/agents/sematext-agent/releasenotes.md [hunk 7]
```diff
@@ -39,6 +39,23 @@ NULL
...
+- Addressed specific scenario where pods were not being monitored when utilizing the Containerd runtime in Kubernetes.
+- Resolved an edge-case bug occurring during discovering services via eBPF, that leads to have incorrect service IP.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:57]: that leads to have incorrect service IP -> which led to incorrect service IP addresses

**Review by cpipilas — CHANGES_REQUESTED** [review]

**Review by boratanrikulu — APPROVED** [review]
LGTM, nice :partying_face:

**Review by cpipilas — APPROVED** [review]

**Pull request merged**